### PR TITLE
Add 2 additional child storage parameters

### LINKED ISSUE #27: Adjust getChildStorage function
The JS API currently doesn't support getting values from child storage through RPC with the latest substrate master.
This PR introduced 2 additional parameters for the childStorage RPC methods, so At the moment, the JS API expects 2, Substrate requires 4 parameters:
https://github.com/paritytech/substrate/pull/4162

I already opened a ticket for that in the 
https://github.com/polkadot-js/api/issues/1679

Once implemented, we need to adjust the`getContractStorage` function.

### PR DESCRIPTION
**Pull request by Stefie**
Fixes https://github.com/paritytech/srml-contracts-waterfall/issues/27

Please only merge once https://github.com/polkadot-js/api/pull/1709 is merged into the Polkadot.js API repo.

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -5,47 +5,45 @@
...
+  dependencies: []
+  interruptible: true
   tags:
     - linux-docker
 
 #### stage:                        build
 
 build-and-test:
-  stage:                           build
-  <<:                              *docker-env
```
Comment (TriplEight) [inline .gitlab-ci.yml:48]: can you please return all changed margings back? It's how this config looks in all the projects...

File: docker-compose.yml [hunk 2]
```diff
@@ -27,7 +28,6 @@ services:
       - "--dev"
       - "--ws-external"
       - "--rpc-external"
-      - "-lruntime=debug"
```
Comment (TriplEight) [inline docker-compose.yml:30]: how this docker-compose is used in gitlab or is it just for local?

Comment (Stefie) [reply docker-compose.yml:30]: only local

Comment (TriplEight) [reply docker-compose.yml:30]: I'll think of how to better automate it there.

Comment (Stefie) [reply docker-compose.yml:30]: Where? No need to automize anything here. That's just my personal docker-compose file that I'm using for development which I just dropped in here. The '- "-lruntime=debug"' was just a leftover from when we were trying to debug an potential memory issue

**Pull request merged**